src/System.ServiceModel.Syndication/ref/System.ServiceModel.Syndication.netcoreapp.cs

```diff
@@ -7,6 +7,10 @@
 
 namespace System.ServiceModel.Syndication
 {
+    public partial class SyndicationFeed
+    {
+        public SyndicationLink Documentation { get{ throw null; } set{ } }
+    }
```

Comment:
Has this been through API review?  If not, we should close the PR until it's been vetted.

Reply:
This has not been approved yet. I added the `NO MERGE` label on the PR for now. I'd like to get feedback from others and see if it's even possible or make sense to add the new APIs before sending out API request..